google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiDetailsPanel.java

```diff
@@ -187,6 +211,9 @@ private void updateUI() {
                 linksTextPane.setText(joinLinks(links));
               });
     }
+
+    apiManagementPanel.setVisible(true);
```

Comment:
Is it hidden by default?

Reply:
it is. that is so that it is not visible when no library is selected in the table. I think a better way would be to make the entire details panel itself hidden until a library is selected. i'll update to that. wdyt?

Reply:
Agree, it will be more consistent.

Reply:
updated